### PR TITLE
feat: Add a static confirm method

### PR DESCRIPTION
**Pull request by liaokaime**

### CODE REVIEW

File: src/ConfirmProvider.js [hunk 1]
```diff
@@ -68,6 +68,13 @@ const buildOptions = (defaultOptions, options) => {
   };
 };
 
+let _confirm
+const confirm = async (options) => {
+    if(_confirm){
+      return await _confirm(options)
+    }
+}
```
Comment (jonatanklosko) [inline src/ConfirmProvider.js:76]:
```suggestion
let confirmGlobal;
```

File: src/ConfirmProvider.js [hunk 2]
```diff
@@ -98,6 +105,12 @@ const ConfirmProvider = ({ children, defaultOptions = {} }) => {
     }
   }, [resolve, handleClose]);
 
+  useEffect(() => {
+      if(confirm){
+        _confirm = confirm
+      }
+  }, [confirm])
```
Comment (jonatanklosko) [inline src/ConfirmProvider.js:112]:
```suggestion
  confirmGlobal = confirm;
```

File: src/ConfirmProvider.js [hunk 3]
```diff
@@ -115,3 +128,4 @@ const ConfirmProvider = ({ children, defaultOptions = {} }) => {
 };
 
 export default ConfirmProvider;
+export {confirm}
```
Comment (jonatanklosko) [inline src/ConfirmProvider.js:131]:
```suggestion
export { confirmGlobal as confirm };
```

**Comment by jonatanklosko** [comment]
Also please run `npm run format` :)

**Comment by liaokaime** [comment]
> Also please run `npm run format` :)

I'm not familiar with github pr, this is my first time using it. In order to format, do I need to create pr again?

**Comment by jonatanklosko** [comment]
@liaokaime no need for a separate PR, you can just run `npm run format` locally`, commit and push. (You also want to `git pull` first to get the changes applied via GitHub UI above :))

**Review by jonatanklosko — APPROVED** [review]

**Comment by jonatanklosko** [comment]
Published in v3.0.10.

**Pull request merged**